compiler/qsc_codegen/src/qir.rs

```diff
@@ -455,7 +546,7 @@ fn get_value_ty(lhs: &rir::Operand) -> &str {
         rir::Operand::Literal(lit) => match lit {
             rir::Literal::Integer(_) => "i64",
             rir::Literal::Bool(_) => "i1",
-            rir::Literal::Double(_) => "f64",
+            rir::Literal::Double(_) => get_f64_ty(),
```

Comment:
What's the value in breaking the pattern of the other types here that all use an inline literal and calling a helper, if that helper just always returns the literal "double"?

Reply:
I broke the pattern to provide documentation for others picking up the code so that they can hover and see why.